src/python/grpcio/grpc/_server.py

```diff
@@ -569,7 +569,7 @@ def _call_behavior(
                         details = (
                             "Calling application raised unprintable Exception!"
                         )
-                        traceback.print_exc()
+                    traceback.print_exc()
                     _LOGGER.exception(details)
```

Comment:
As discussed, let's also log this via the `_LOGGER`. This will ensure that the message shows up:

1. On STDERR when no logging sink has been configured.
2. In the log file, if it is not combined with STDERR.

Reply:
Sure, since we're already logging printable exceptions, I'll just add `_LOGGER` support for un-printable exceptions.